pkg/security/scc/byrestrictions.go

```diff
@@ -15,37 +15,50 @@ func (s ByRestrictions) Less(i, j int) bool {
 	return pointValue(s[i]) < pointValue(s[j])
 }
 
+// The following constants define the weight of the restrictions and used for
+// calculating the weight of the particular SCC. The lower the number, the more
+// restrictive SCC is. Make sure that weak restrictions are always valued
+// higher than the combination of the strong restrictions.
+
+const privilegedWeight int = 20
```

Comment:
that seems a little...excessive?  Why use an enum type here other than being able to type it somewhere?